sites/geohub/src/components/MapQueryInfoControl.svelte

```diff
@@ -501,4 +501,10 @@
     background-size: 75% !important;
     background: url('data:image/svg+xml;charset=UTF-8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!-- Font Awesome Pro 5.15.4 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) --><path d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z"/></svg>');
   }
+  :global(.maplibregl-popup-close-button) {
+    top: 15px !important;
+    right: 10px !important;
+    height: 30px;
+    width: 30px;
```

Comment:
Looks like having no change of size of closing button. This x button uses just `x` character. Could you try to use `font-size` instead of height and width?

Reply:
@JinIgarashi I have added the `font-size` css property. The `height` and `width` needs to be there also so the background of the button to cover the height of the `QueryInformation` header

![screenshot](https://user-images.githubusercontent.com/45137948/210552412-5b6bb7e5-be12-48cc-bfdc-7321b6fa8c13.png)